learn/quickstart/operators_control_flow.md

```diff
@@ -119,19 +119,104 @@ __Example:__ `do` loop with skip
 
 ```fortran
   integer :: i
-  do i=1,10,2    
+  do i=1,10,2
     print *, i   ! Print odd numbers
   end do
 ```
 
+### `do while()`
 
-__Example:__ `do while` loop
+A condition may be added to a `do` loop with the `while` keyword. The loop will be executed while the condition given
+in `while()` evaluates to `.true.`.
+
+__Example:__ `do while()` loop
 
 ```fortran
   integer :: i
   i = 1
-  do while (i<11)   
+  do while (i<11)
     print *, i
     i = i + 1
   end do
-```
\ No newline at end of file
+  ! Here i = 11
+```
+
+### `do concurrent()`
+
+The `do concurrent` loop is used to specify that the _inside of the loop has no interdependencies_. By this, it is indicated
+that any loop does not depend on the loops before it. It is also necessary that any state changes that may occur must only
+happen within each `do concurrent` loop. Because of these requirements, it is important to be carful with what is written inside the loop.
```

Comment:
```suggestion
The `do concurrent` loop is used to explicitly specify that the _inside of the loop has no interdependencies_; this informs the compiler that it may use parallelization/_SIMD_ to speed-up execution of the loop and conveys programmer intention more clearly. More specifically, this means
that any loop iteration does not depend on the prior execution of other loop iterations. It is also necessary that any state changes that may occur must only happen within each `do concurrent` loop. 
These requirements place restrictions on what can be placed within the loop body.
```